forum/service/src/main/java/org/exoplatform/forum/service/search/DiscussionSearchConnector.java

```diff
@@ -77,9 +93,9 @@ public DiscussionSearchConnector(InitParams initParams, DataStorage storage) {
         Topic topic = storage.getTopicByPath(id.getTopicPath(), false);
         StringBuilder sb = new StringBuilder();
         sb.append(forum.getForumName());
-        sb.append(" - " + topic.getPostCount() + " replies");
+        sb.append(" - " + topic.getPostCount() + " " + (resourceBundle != null ? resourceBundle.getString("DiscussionSearchConnector.reply") : "replies"));
```

Comment:
It should be a key with parameter: `DiscussionSearchConnector.reply={0} replies`